pkg/config/data.go

```diff
@@ -167,13 +169,16 @@ func (d *Data) Endpoint() (string, Source) {
 	return DefaultEndpoint, SourceDefault // this method should not fail
 }
 
+// FileName is the name of the application configuration file.
+const FileName = "config.toml"
+
 // FilePath is the location of the fastly CLI application config file.
 var FilePath = func() string {
 	if dir, err := os.UserConfigDir(); err == nil {
-		return filepath.Join(dir, "fastly", "config.toml")
+		return filepath.Join(dir, "fastly", FileName)
 	}
 	if dir, err := os.UserHomeDir(); err == nil {
-		return filepath.Join(dir, ".fastly", "config.toml")
+		return filepath.Join(dir, ".fastly", FileName)
```

Comment:
There's a few places in the code where I need to reference the string `"config.toml"` so it made sense to move it to a package level constant.